pages/nft-book-store/status/[classId].vue

```diff
@@ -172,15 +172,14 @@
             <h3 class="font-bold font-mono">
               Editions
             </h3>
-            <div class="flex justify-center">
+            <div class="flex justify-between items-center gap-4">
+              <div>
+                Unassigned Stock : {{ UnassignedStock }}
```

Comment:
How about manually delivered edition? We would have unsold stock, sold but pending deliver stock, and real unassigned stock

Reply:
[🐛 Update stock calculation logic](https://github.com/likecoin/nft-book-press/pull/334/commits/d3c1bbd3568101331a857745b30f07767e0abf84)